CHANGELOG.md

```diff
@@ -19,8 +19,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Hardware support for:
   - Vector single-width floating-point add/subtract instructions (vfadd, vfsub, vfrsub)
+  - Vector widening floating-point add/subtract instructions (vfwadd, vfwsub, vfwadd.w, vfwsub.w)
   - Vector single-width floating-point multiply instructions (vfmul)
+  - Vector widening floating-point multiply instructions (vfwmul)
   - Vector single-width floating-point fused multiply-add instructions (vfmacc, vfmadd)
+  - Vector widening floating-point fused multiply-add instructions (vfwmacc, vfwnmacc, vfwmsac, vfwnmsac)
```

Comment:
Please move this to the Unreleased section. Please also update the `FUNCTIONALITIES` file.